packages/sdk/client-services/src/packlets/invitations/invitations-handler.ts

```diff
@@ -287,19 +308,15 @@ export class InvitationsHandler {
                 invitation.spaceKey = introductionResponse.spaceKey;
               }
 
-              // TODO(dmaretskyi): Add a callback here to notify that we have introduced ourselves to the host (regardless of auth requirements).
-
               // 2. Get authentication code.
-              // TODO(burdon): Test timeout (options for timeouts at different steps).
               if (isAuthenticationRequired(invitation)) {
-                stream.next({ ...invitation, state: Invitation.State.AUTHENTICATING });
-
                 for (let attempt = 1; attempt <= MAX_OTP_ATTEMPTS; attempt++) {
```

Comment:
MAX might come from a default options object in the function (so we can control better in tests?)